android/src/main/java/io/card/react/ReactCardIOModule.java

```diff
@@ -69,6 +69,11 @@ public void scan(ReadableMap options, Promise promise) {
 
       getReactApplicationContext().startActivityForResult(scanIntent, REQUEST_CARD_SCAN, null);
     }
+    
+    public void onActivityResult(android.app.Activity activity, int requestCode, int resultCode, Intent intent) {
```

Comment:
You can just use `...Result(Activity activity, int request...` instead of `android.app.Activity`.